lib/fieldhand/record.rb

```diff
@@ -14,6 +14,8 @@ class Record
     def initialize(element, response_date = Time.now)
       @element = element
       @response_date = response_date
+
+      Ox.default_options = { :encoding => 'utf-8' }
```

Comment:
As this will override Ox's options globally (including if a user has set this themselves), perhaps we should prefer being explicit when we use `Ox.dump` instead to localise the change?

```ruby
Ox.dump(element, :encoding => 'utf-8')
```